app/views/shared/_footer.html.erb

```diff
@@ -8,7 +8,7 @@
                             <i class="fas fa-map-marker-alt"></i>
                             <div class="cta-text">
                                 <h4>Address</h4>
-                                <span>212 Bd. de la Tunisie, Casablanca</span>
```

Comment:
Hey Ismail, what you did is wrong, It's not 220, it's actually 221. Let's change it please

Reply:
Hey Said, I just modified it and pushed it. Thanks for pointing this out!